### PR TITLE
[+] Friendly URLs - Ensure that projects names are unique

### PR DESCRIPTION
**Pull request by slimee**
Now, we block project creation in case of Conflict.
This is due to the project name unicity constraint.

### CODE REVIEW

**Comment by slimee** [comment]
[URLs - (Lumber-forestadmin) Ensure that projects names are unique](https://trello.com/c/9J4HHW6e/3687-urls-lumber-forestadmin-ensure-that-projects-names-are-unique)

**Comment by slimee** [comment]
Note: If the user create a project with a previous version of Lumber(forest-admin) after "Project names unicity" release on the server, it will fail with an NPE instead of a good error message.

**Comment by slimee** [comment]
ADHERE WITH https://github.com/ForestAdmin/forestadmin-server/pull/793

File: services/api.js [hunk 1]
```diff
@@ -33,32 +33,13 @@ function API() {
...
-    try {
-      newProject = await agent
+    const newProject = await agent
         .post(`${this.endpoint}/api/projects`)
         .set('forest-origin', 'Lumber')
         .set('Content-Type', 'application/json')
         .set('Authorization', `Bearer ${sessionToken}`)
         .send(new ProjectSerializer(project))
         .then(response => ProjectDeserializer.deserialize(response.body));
-    } catch (error) {
```
Comment (larcin) [inline services/api.js]: [Misc] Why did you remove the try catch here exactly ?

Comment (slimee) [reply services/api.js]: We don't make two calls in case of 'Conflict' anymore. Now it's handled at the upper level with just a log.
